### PR TITLE
added AddToDoForm, destructured props

### PR DESCRIPTION
**Pull request by SergeDot**

### CODE REVIEW

File: src/TodoList.jsx [hunk 1]
```diff
@@ -1,33 +1,14 @@
...
-    country: "UK"
-  }];
+const TodoList = ({todoList}) => {
 
   return (
     <ul>
-      {todoList.map(item => {
-        return <TodoListItem todo={item} key={item.id} />
+      {todoList.map(({id, ...item}) => {
+        return <TodoListItem key={id} {...item} />
```
Comment (riousenkai) [inline src/TodoList.jsx]: Let's not spread item here (and on the map). It's easier to read if you assign title to its own prop.

Remember that readability is important. Code is read and tweaked more than it's written. Readable code also cuts down the learning curve for newbies and makes it easy for others to get what's going on and make changes.

File: src/App.jsx [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
-import AddTodoForm from './AddTodoForm'
-import { useState } from 'react'
+import './App.css';
+import TodoList from './TodoList';
+import AddTodoForm from './AddTodoForm';
+import { useState } from 'react';
 
 function App() {
-  const [newTodo, setNewTodo] = useState('')
+  const [newTodo, setNewTodo] = useState('');
```
Comment (riousenkai) [inline src/App.jsx]: Is this still needed?

Comment (SergeDot) [reply src/App.jsx]: > Is this still needed?

No. I removed it and the line that used it. The state is not updated anywhere for this var.

File: src/TodoList.jsx (hunk 1 above)
Comment (SergeDot) [reply src/TodoList.jsx]: > Let's not spread item here (and on the map). It's easier to read if you assign title to its own prop.

Got it, thank you, redestructured it.

**Review by riousenkai — APPROVED** [review]

**Comment by riousenkai** [comment]
Please merge this PR and update your next assignment's code

**Pull request merged**